python_client/tests/test_bags.py

```diff
@@ -12,6 +12,13 @@ def test_can_get_bag(client):
     assert len(resp["manifest"]["files"]) == 3
 
 
+def test_can_get_bag_version(client):
+    resp = client.get_bag(space_id="alex-testing", source_id="b12345", version="v1")
+    assert resp["space"]["id"] == "alex-testing"
+    assert resp["id"] == "alex-testing/b12345"
+    assert len(resp["manifest"]["files"]) == 3
```

Comment:
What about a test for `v2` of this bag, to check that if you request a version that doesn't exist, it returns correctly?

And maybe a test for a `v3` that does exist, and asserting it returns the correct version?

Reply:
I've added a test for missing bag (since it does trigger a different code path here). I'm not sure what a test for v3 would add given that the client has no logic for returning the correct version and relies on the server to do so, but it's possible I'm missing something here :)

Reply:
If the storage service only knows about `v1`, you'll get an identical response for `/bag` and `/bag?version=v1`. The idea of a test for v3 is to make sure the client is actually adding that `?version` parameter.

Reply:
That makes sense :-) Ready for re-review!